homeassistant/components/jellyfin/media_source.py

```diff
@@ -155,10 +154,7 @@ async def _build_library(
             return await self._build_music_library(library, include_children)
         if collection_type == COLLECTION_TYPE_MOVIES:
             return await self._build_movie_library(library, include_children)
-        if collection_type == COLLECTION_TYPE_TVSHOWS:
```

Comment:
Why have you removed TVSHOWS type inside a tests PR?

Reply:
This was to remove unreachable code. The raise BrowseError that was here before could not be reached because the supported collection types have been filtered out earlier. There is no change in functionality.